.github/workflows/pr-build.yml

```diff
@@ -1,45 +1,122 @@
 name: PR Build
 
 on:
-  pull_request:
-    paths: 
-      - ".github/workflows/pr-build.yml"
-      - "android/**"
-      - "assets/**"
-      - "lib/**"
-      
+  workflow_dispatch:
+    inputs:
+      # Flutter
+      flutter-branch:
+        description: Flutter branch
+        type: choice
+        default: 'stable'
+        options: 
+          - stable
+          - beta
+          - dev
+          - master
+      flutter-cache:
+        description: Cache
+        type: boolean
+        default: true
```

Comment:
Why is this necessary?

Reply:
I added this in case someone finds it useful because sometimes I get into a problem where I need to remove the Flutter cache to build.